xdsl/dialects/csl/csl.py

```diff
@@ -1255,7 +1257,7 @@ class IncrementDsdOffsetOp(IRDLOperation):
     name = "csl.increment_dsd_offset"
 
     op = operand_def(DsdType)
-    offset = operand_def(i16_value)
+    offset = operand_def(eq(i16) | eq(i16_value))
```

Comment:
The value naming is confusing, can you please rename this type to be consistent with `i16` and the way it's printed in the IR?
```suggestion
    offset = operand_def(eq(i16) | eq(si16))
```

Reply:
It makes it consistent with the cerebras docs.